graylog2-web-interface/src/components/cluster/GraylogClusterOverview.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Reflux from 'reflux';
+import { Row, Col } from 'react-bootstrap';
+
+import { Spinner } from 'components/common';
+
+import StoreProvider from 'injection/StoreProvider';
+const NodesStore = StoreProvider.getStore('Nodes');
+
+const GraylogClusterOverview = React.createClass({
+  mixins: [Reflux.listenTo(NodesStore, '_onNodesChange', '_onNodesChange')],
+
+  getInitialState() {
+    return {
+      clusterId: undefined,
+      nodeCount: 0,
+    };
+  },
+
+  _onNodesChange() {
+    this.setState({ clusterId: NodesStore.getClusterId(), nodeCount: NodesStore.getNodeCount() });
```

Comment:
Why are the cluster id and the node count not included in the triggered state of the store, so we can just `connect()` to the store? We would also save those getters in the `NodesStore` which are kinda against Reflux principles (at least if they're avoidable).